openssl-sys/src/handwritten/x509v3.rs

```diff
@@ -96,12 +96,20 @@ extern "C" {
         indent: c_int,
     ) -> c_int;
 
-    #[cfg(ossl110)]
+    #[cfg(ossl111)]
```

Comment:
Why? These are definitely present in 1.0.0.

Reply:
yes, I will update the code later and use the new added Asn1OctetString